icubam/backoffice/handlers/base.py

```diff
@@ -30,7 +30,7 @@ def render_list(
     self, data, objtype, create_handler=None, upload=False, **kwargs
   ):
     route = None if create_handler is None else create_handler.ROUTE
-    upload_type = route if upload else None
+    upload_type = objtype if upload else None
```

Comment:
i don t understand what motivated this change.. can you give an example?

Reply:
The motivation was in the corresponding commit message. This confusion between route and object type made it impossible to upload data which does not have a create_handler.